inc/commondbtm.class.php

```diff
@@ -2313,8 +2319,8 @@ function showFormButtons($options = []) {
     * @param array   $options Array of possible options:
     *     - withtemplate : 1 for newtemplate, 2 for newobject from template
     *
-    * @return integer|void value of withtemplate option (exit of no right)
-   **/
+    * @return mixed|string
```

Comment:
It returns the value of option `withtemplate` or empty string. So documentation should tell the returned value is `integer|string` and that it is corresponding to `withtemplate` option value.